commands/pipeline/run/pipeline_run.go

```diff
@@ -57,17 +63,30 @@ func NewCmdRun(f *cmdutils.Factory) *cobra.Command {
 				return err
 			}
 
-			// TODO: support setting pipeline variables via cli.
-			v := []*gitlab.PipelineVariable{
+			pipelineVars := []*gitlab.PipelineVariable{
 				{
 					Key:          "GLAB_CLI_KEY",
 					Value:        "GLAB_CLI_VAL",
 					VariableType: "env_var",
 				},
 			}
```

Comment:
I don't think this should be there anymore. You can initialize as an empty slice

```go
pipelineVars := []*gitlab.PipelineVariable{}
```

Reply:
Okay, didn't know if this is needed so just left it there. Now is removed